js/preload/preloadMain.js

```diff
@@ -12,6 +12,7 @@
 // https://github.com/electron/electron/issues/2984
 //
 const { ipcRenderer, remote } = require('electron');
+const _ = require('underscore');
```

Comment:
Sure. Makes sense. The idea behind using underscore was to keep it ready if it was required in the future. I've changed it now.